facility-types.jsonld

```diff
@@ -197,15 +192,17 @@
     {
       "type": "Concept",
       "id": "https://openactive.io/facility-types#1204c026-b2db-4f2f-b0c5-21a7f41f7284",
-      "prefLabel": "Racquetball Court"
+      "prefLabel": "Racquetball Court",
+      "activity": [
+        "https://openactive.io/activity-list#32225241-1c8c-4ba3-b540-22b710433c18"
+      ]
     },
     {
       "type": "Concept",
       "id": "https://openactive.io/facility-types#0e847c2f-d3c9-4c89-9fc1-43fb82de4395",
       "prefLabel": "Rugby Pitch",
       "activity": [
-        "https://openactive.io/activity-list#9e143963-8f17-41ab-8915-3828c5a9ddae",
-        "https://openactive.io/activity-list#a7f0a364-a496-4446-aec6-594a629353b3"
```

Comment:
Replace Rugby Union+Rugby League with Rugby

Reply:
My concern here would be for sessions already mapped to this facility type for league/union will no longer be valid. What's the reasoning behind removing these rather than just adding Rugby?

Reply:
@nathansalter You're totally right, removing this was overzealous, I think there is an exact equivalence between Rugby Pitch and Rugby Union/League as there are no specific Union/League pitches.